package.json

```diff
@@ -38,20 +53,35 @@
     "grunt-lintspaces": "^0.7.0",
     "grunt-merge-json": "^0.9.5",
     "grunt-webpack": "^1.0.11",
-    "jscs-jsdoc": "shaoshing/jscs-jsdoc#7fbf64b",
+    "jscs-jsdoc": "^1.2.0",
     "jscs-trailing-whitespace-in-source": "0.0.1",
     "json-loader": "^0.5.3",
-    "jsx-loader": "^0.13.2",
     "less": "^2.5.3",
-    "less-loader": "^2.2.1",
+    "lintspaces": "^0.3.3",
     "loader-utils": "^0.2.11",
-    "lodash": "^3.10.1",
-    "node-lintspaces": "shaoshing/node-lintspaces",
+    "react-addons-perf": "^0.14.2",
     "react-tools": "^0.13.1",
     "uglifyjs": "^2.4.10",
     "url-loader": "^0.5.6",
     "webpack": "^1.12.3",
-    "webpack-dev-server": "^1.12.1",
+    "webpack-dev-server": "^1.12.1"
+  },
+  "dependencies": {
+    "bluebird": "^3.0.5",
+    "classnames": "^2.2.0",
+    "d3": "^3.5.9",
+    "fluxxor": "^1.7.3",
+    "generator-connection": "adobe-photoshop/generator-connection",
+    "immutable": "^3.7.5",
+    "lodash": "^3.10.1",
+    "loglevel": "^1.4.0",
+    "mathjs": "^2.4.2",
+    "react": "^0.14.2",
+    "react-addons-pure-render-mixin": "^0.14.2",
+    "react-dom": "^0.14.2",
+    "scriptjs": "^2.5.8",
+    "spaces-adapter": "adobe-photoshop/spaces-adapter.git",
+    "tinycolor2": "^1.1.2",
```

Comment:
Can we alias this to `tinycolor` so that we don't have to change all the require calls?